src/main/java/lotto/automatic/LottoGame.java

```diff
@@ -2,30 +2,29 @@
 
 import lotto.automatic.domain.*;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class LottoGame {
 
-    private final LottoGenerator generator;
+    public List<Lotto> generateLotto(int lottoAmount) {
 
-    public LottoGame() {
+        List<Lotto> list = new ArrayList<>();
 
-        generator = new LottoGenerator(new RandomShuffle());
-    }
+        for(int i = 0; i < lottoAmount; i ++ ) {
+            list.add(Lotto.auto());
+        }
 
-    public List<Lotto> generateLotto(int lottoAmount) {
-        return generator.generateLotto(lottoAmount);
+        return list;
     }
 
     public LottoResult getLottoResult(int investMoney, List<LottoRank> rankList) {
 
         return new LottoResult(investMoney, rankList);
     }
 
-    public List<LottoRank> getLottoRank(List<Lotto> lottoList, List<LottoNumber> lottoNumbers) {
-        Lotto winningLotto = new Lotto(lottoNumbers);
-
-        LottoRankCollector collector = new LottoRankCollector(lottoList, winningLotto);
+    public List<LottoRank> getLottoRank(List<Lotto> lottoList, WinningLotto winningLottoNums) {
```

Comment:
```suggestion
    public List<LottoRank> getLottoRank(List<Lotto> lottoList, WinningLotto winningLotto) {
```
오타인듯 해요. 
추가로 List<LottoRank> 는 구입한 로또와 당첨 로또를 가지고 결과를 도출해 내고 있으니 LottoRankCollector 클래스를 사용하는것 보다는 WinningLotto 에서 결과값을 전달해주는것이 자연스럽지 않을까요 ?

Reply:
RankCollector 대신 WinningLotto 에서 결과값을 가져올 수 있도록 개선하였습니다.
RankCollector 가 필요없어졌네요.